stratum/hal/lib/bcm/bcm_chassis_manager_test.cc

```diff
@@ -1569,7 +1579,7 @@ TEST_P(BcmChassisManagerTest,
   }
   // Verify config.bcm in this case. Logical ports start from 1 and go and up.
   std::string bcm_sdk_config;
-  /* SDK6 Only
+  /* BCM SDK Only
```

Comment:
Should we put these sections back? Okay to put a todo or open an issue.

Reply:
These test the sdk config file generation. It has been moved into the SdkWrapper which don't have unit tests.

Reply:
fair enough